packages/apps/postgres/README.md

```diff
@@ -66,44 +66,57 @@ See:
-| `bootstrap.oldName`      | Name of database cluster before deleting                                                                             | `""`    |
+| Name                     | Description                                                | Type     | Value  |
+| ------------------------ | ---------------------------------------------------------- | -------- | ------ |
+| `backup`                 | Backup configuration                                       | `object` | `null` |
+| `backup.enabled`         | Enable regular backups                                     | `bool`   |        |
+| `backup.schedule`        | Cron schedule for automated backups                        | `string` |        |
+| `backup.retentionPolicy` | Retention policy                                           | `string` |        |
+| `backup.destinationPath` | Path to store the backup (i.e. s3://bucket/path/to/folder) | `string` |        |
+| `backup.endpointURL`     | S3 Endpoint used to upload data to the cloud               | `string` |        |
+| `backup.s3AccessKey`     | Access key for S3, used for authentication                 | `string` |        |
+| `backup.s3SecretKey`     | Secret key for S3, used for authentication                 | `string` |        |
+
+### Bootstrap parameters
+
+| Name                     | Description                                                                                                          | Type        | Value   |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------- | ----------- | ------- |
+| `bootstrap`              | Bootstrap configuration                                                                                              | `object`    | `null`  |
+| `bootstrap.enabled`      | Restore database cluster from a backup                                                                               | `bool`      |         |
+| `bootstrap.recoveryTime` | Timestamp (PITR) up to which recovery will proceed, expressed in RFC 3339 format. If left empty, will restore latest | `string`    |         |
+| `bootstrap.oldName`      | Name of database cluster before deleting                                                                             | `string`    |         |
+| `resources`              | Resources                                                                                                            | `object`    | `{}`    |
+| `resources.cpu`          | CPU                                                                                                                  | `*quantity` |         |
+| `resources.memory`       | Memory                                                                                                               | `*quantity` |         |
```

Comment:
Very cool, we can now describt variables in objects without explicitly defining them in values.yaml.